docs/user-guide/configure-certificates.md

```diff
@@ -1,130 +1,123 @@
 
-* If you are using a USS keystore, then the script zowe-setup-certificates.env is the configuration step required to create the USS directory that contains the certificate. This is described in detail in Configuring Zowe certificates in a USS KeyStore.
+### File-based (PKCS12) certificate setup
 
-* If you are using a key ring, the sample JCL member `ZWEKRING` provided in the PDS library `SZWESAMP` contains the security commands to create a key ring and manage its associated certificates. This is described in Configuring Zowe certificates in a key ring which provides instructions for how to configure Zowe to work with the following certificates.
-    * a self-signed certificate
-    * a certificate signed with an existing certificate authority
-    * an existing certificate already held in the SAF database that can be added to the Zowe key ring.
+Zowe is able to use PKCS12 certificates that are stored in USS. Zowe uses a `keystore` directory to contain its certificates primarily in PKCS12 (`.p12`, `.pfx`) file format, but also in PEM (`.pem`) format. The truststore is in the `truststore` directory that holds the public keys and CA chain of servers which Zowe communicates with (for example z/OSMF).
 
-For both scenarios, where the certificate is held in a USS Java Keystore or a z/OS key ring, the `USS KEYSTORE_DIRECTORY` is still required which is created with the script zowe-setup-certificates.sh.
+### z/OS key ring-based certificate setup
 
-* In the USS scenario, this directory holds the `.cer` and `.pem` files for the certificate itself.
-* In the key ring scenario, this directory stores the location and name of the Zowe key ring and its certificates.
+Zowe is able to work with certificates held in a **z/OS Key ring**.  
 
-### Enable certificate validation without hostname validation
+The JCL member `.SZWESAMP(ZWEKRING)` contains security commands to create a SAF key ring. By default, this key ring is named `ZoweKeyring`. You can use the security commands in this JCL member to generate a Zowe certificate authority (CA) and sign the server certificate with this CA. The JCL contains commands for three z/OS security managers: RACF, TopSecret, and ACF2.
 
-To enable certificate validation without hostname validation, set `zowe.verifyCertificates: NONSTRICT`. Using this setting, the certificate Common Name or Subject Alternate Name (SAN) is not checked. Disabling this parameter facilitates deployment to environments where certificates are valid but do not contain a valid hostname. This configuration is for development purposes only and should not be used for production.
+There are two ways to configure and submit `ZWEKRING`:
 
-A proper setup of the truststore is mandatory to successfully start Zowe with `zowe.verifyCertificates: STRICT`.
+- Customize and submit the `ZWEKRING` JCL member.
```

Comment:
I believe Joe's comment applies here too?

Reply:
I edited this to:
"Copy the JCL `ZWEKRING` member and customize its values."

Please check for accuracy.